package.json

```diff
@@ -20,16 +20,16 @@
     "cf-unlink": "node scripts/npm/unlink"
   },
   "dependencies": {
-    "cf-buttons": "^3.2.0",
-    "cf-core": "^3.5.0",
-    "cf-expandables": "^3.1.0",
-    "cf-forms": "^3.1.2",
-    "cf-grid": "^3.2.0",
-    "cf-icons": "^3.2.1",
-    "cf-layout": "^3.2.2",
-    "cf-pagination": "^3.1.1",
-    "cf-tables": "^1.1.0",
-    "cf-typography": "^3.1.0",
+    "cf-buttons": "^4.1.0",
+    "cf-core": "^4.1.0",
+    "cf-expandables": "^4.1.0",
+    "cf-forms": "^4.1.0",
+    "cf-grid": "^4.1.0",
+    "cf-icons": "^4.1.0",
+    "cf-layout": "^4.1.0",
+    "cf-pagination": "^4.1.0",
+    "cf-tables": "^4.1.0",
+    "cf-typography": "^4.1.0",
```

Comment:
Several of these need to get updated to their latest releases. Easiest thing to reference is the badges at the top of the capital-framework README.